### PR TITLE
Add phpunit to composer.json as require-dev

### PR DESCRIPTION
**Pull request by padraic**
Set to version 4.4 or greater and no issues noted. This allows tests to be run using vendor/bin/phpunit "../vendor/bin/phpunit" from ./tests.

### CODE REVIEW

**Comment by thornag** [comment]
@cordoval Nah, I don't assume anything, I think you assumed that I did ;)

I'm speaking from the perspective on an in-house software team, all of our packages are structured this way, there is tendency to use phpspec over phpunit - as such, phpunit is not that much of a given for us.

Having preference to to always run containers from the same image we don't have to worry to provide dev requirements for all different scenarios or manage multiple images - we simply run prod like.

The argument with shell would be valid if a project or package was indeed relying on it and it's different functionality, then I would also look for a way to either lock it in or develop for all versions.

I think there is too much made out of this discussion, I spill think about phpunit same way I do about mockery, phpspefc, behat or anything else.

So to recap, I agree with your argument and see it as valid - matter of preference perhaps?

**Comment by cordoval** [comment]
do it with any other library, but perhaps not this one :baby: that should close this ticket

**Comment by padraic** [comment]
Does nobody relax over the Holidays anymore ;). I thought I'd get to March before responses! Closing PR based on discussion - It's not critical to have this, and there are no PHPUnit version boundary issues that I know of...here. In other places, unfortunately yes but that's neither here nor there.
